manifests/current.pp

```diff
@@ -93,6 +97,11 @@
     }
   }
 
+  file { "/etc/${package_name}/tags.yaml":
```

Comment:
From what I've heard it will be picked up by the agent, but not directly. If you want it directly you have to unregister/register the client again. That is for the file-redaction.yaml file.